.hemtt/project.toml

```diff
@@ -0,0 +1,32 @@
+name = "Zeus Enhanced"
+prefix = "zen"
+author = "ZEN Team"
+mainprefix = "x"
+
+[files]
+include = [
+    "mod.cpp",
+    "logo_zen_ca.paa",
+    "logo_zen_small_ca.paa",
+    "LICENSE",
+    "AUTHORS.txt",
+    "README.md",
+    "meta.cpp"
+]
+
+[asc]
+enabled = true
+exclude = [
+    ".inc.sqf",
+    "/dev/"
+]
+
+key_name = "{{prefix}}_{{version}}"
+authority = "{{prefix}}_{{version}}-{{git \"id 8\"}}"
```

Comment:
Does HEMTT still use these?

Reply:
@BrettMayson

Reply:
No, neither does it use `[header_exts]` section or `releasebuild` field.

Reply:
Should it be replaced by anything or just dropped?

Reply:
Dropped. HEMTT v1 enforces a standard.